<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TgsWid</class>
 <widget class="QWidget" name="TgsWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="GaugeArc2" name="gaugeArc2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="title">
    <string>通过率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>300</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试设备数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试通过数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试失败数量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>设备选择</string>
   </property>
   <widget class="QLineEdit" name="codeEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="codeRadio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>141</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>查询工装条码</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="allRadio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>查询所有</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>601</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>时间选择</string>
   </property>
   <widget class="QRadioButton" name="trRadio">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>查询时间段</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="startDateEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="endDateEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="todayRadio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>115</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>查询今天</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="todayDateEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="numLcd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="okLcd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="errLcd">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GaugeArc2</class>
   <extends>QWidget</extends>
   <header>gaugearc2.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
